Tests/Utils/Providers/AwsCredentialsProvider.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using AmazonGameLift.Editor;
+using Moq;
+
+namespace AmazonGameLiftPlugin.Editor.UnitTests
+{
+    public class AwsCredentialsProvider
```

Comment:
Maybe we should rename this to `AwsCredentialsTestProvider` or similar since `AwsCredentialsProvider` is an actual thing in the AWS space so the code is a little more confusing. That was my bad on the original naming suggestion

Reply:
Makes sense, Renamed.